package.json

```diff
@@ -33,21 +34,25 @@
     "url": "https://github.com/agenda/agenda/issues"
   },
   "dependencies": {
-    "cron": "~1.1.0",
+    "cron": "~1.2.1",
     "date.js": "~0.3.1",
-    "debug": "^2.6.8",
+    "debug": "~3.0.0",
```

Comment:
You can get rid of these changes by rebasing with master `git rebase origin/master`